src/js/components/Form/Form.js

```diff
@@ -339,8 +339,7 @@ const Form = forwardRef(
           let result;
           if (
             required &&
-            // false is for CheckBox
-            (value2 === undefined || value2 === '' || value2 === false)
+            (!value2 || (Array.isArray(value2) && !value2.length))
```

Comment:
is there a reason we needed to remove the existing logic as opposed to just adding `(Array.isArray(value2) && !value2.length)` as one more condition? just worried we may be losing something. otherwise this is looking good.

Reply:
@taysea there's not a good reason, to be honest, I just thought the logic could be simplified.

This verification:
```js
if (!value) {
  // do something
}
```
is equal to:
```js
if (value === null || value === undefined || value === '' || value === false) {
  // do something
}
```

But I can bring back the old logic, what do you think?

Reply:
We still need to accommodate when `value` is set to 0. That will not be correctly handled by `!value`.

Reply:
^ Good catch, thanks for that. In this case, I would recommend leaving the existing logic and just adding your array condition as another `||` @Lukazovic

Reply:
Good catch @ericsoderberghp! I didn't think about that, thanks.

I'll update it, @taysea. Thanks